web/modules/custom/routing/src/Controller/RoutingController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Drupal\routing\Controller;
+
+use Drupal\Core\Controller\ControllerBase as ControllerControllerBase;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Custom controller named as defined in the info.yml file
+ * Implements a page with a custom message containing username
+ */
+class RoutingController extends ControllerControllerBase {
+
+  /**
+   * Class property for current user
+   * @var \Drupal\Core\Session\AccountInterface
```

Comment:
This commenting style is wrong.
Same for other comments you have given.
Please check for examples from the class files in core/ folder.

Reply:
@vivekpan27 I am unable to figure out the correct style in this scenario. Can you please provide some insight or an example that how this comment can be implemented

Reply:
Check in any other class file.
You have to give proper indentation, line gapes, full stops etc.